### DISCUSSION TITLE
Does anyone know what configuration I need (maybe a config template someone has) to mimic phpstorm's default formatting for javascript?

### DISCUSSION THREAD

**Discussion by ivan006** [prettier/prettier#13629] [Q&A]
**Does anyone know what configuration I need (maybe a config template someone has) to mimic phpstorm's default formatting for javascript?**

I just started at a company where the lead dev uses phpstorm and i wanna introduce them to prettier but in the most non-intrusive way possible. I was think we could start using prettier exactly the same way phpstorm works for js then at least I can work on vscode and format my code the way the lead devs wants it.

**Comment by kachkaev** [comment]
Hi @ivan006! It's best to use Prettier without any configuration options. This way your code style is implicitly syncs with code style in thousands of popular repos. The initial diff may be significant, but it’s worth it in the long term!
